releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- The vectorized execution engine now supports `bool_and`/`bool_or` builtin aggregation functions. [#44164][#44164]
+
+### Command-line changes
+
+- Connections using Unix sockets are now accepted even when the server is running in secure more. (Consult [`cockroach start --help`](../v20.1/cockroach-start.html) for details about the `--socket` parameter.) [#43848][#43848]
+- The [`cockroach init`](../v20.1/cockroach-init.html) command now waits until the node at the provided server address is ready to accept initialization. This also waits for network readiness. This makes it easier to implement initialization scripts by removing the need for a loop. (Such a loop was operationally unsafe anyway and should be discouraged for correctness.) [#43904][#43904]
+- The MovR dataset will now be split among all nodes in the demo cluster. [#43798][#43798]
+- [`cockroach demo --with-load`](../v20.1/cockroach-demo.html) can now round robin queries to all nodes in the demo cluster. [#43474][#43474]
+- The SQLSmith workload now accepts an argument `error-sensitivity` which controls upon what types of errors the workload exits on. [#43925][#43925]
+- [`cockroach gen haproxy`](../v20.1/cockroach-gen.html) now excludes decommissioned nodes. [#43908][#43908]
+- The CLI commands [`cockroach node decommission`](../v20.1/cockroach-node.html#node-decommission) and [`cockroach node recommission`](../v20.1/cockroach-node.html#node-recommission) now produce a warning on the standard error if one of the node(s) specified is already (d/r)ecommissioned. [#43915][#43915]
+- The CockroachDB node command ([`start`](../v20.1/cockroach-start.html)/[`start-single-node`](../v20.1/cockroach-start-single-node.html)) no longer initiates a 1-minute hard shutdown countdown after a request to gracefully terminate. This means that graceful shutdowns are now free to take longer than one minute. It also means that deployments where a maximum shutdown time must be enforced must now use a service manager that is suitably configured to do so. [#44074][#44074]
+- Three new CLI commands `cockroach auth-session login`, `cockroach auth-session list` and `cockroach auth-session logout` are now provided to facilitate the management of web sessions. The command `auth-session login` also produces a HTTP cookie which can be used by non-interactive HTTP-based database management tools. It also can generate such a cookie for the `root` user, who would not otherwise be able to do so using a web browser. [#43872][#43872]
+
+### Admin UI changes
+
+- Decommissioned node history page is now viewable on a dedicated page. This reduces the amount of information on the [Cluster Overview](../v20.1/admin-ui-cluster-overview-page.html) page [#42817][#42817]
```

Comment:
No need for first use of "page" in this sentence

Reply:
This and several other sentences in this block need periods at the end.  (Probably worth checking the whole doc - I did not catch all of them, I'm sure.)